_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: A year of progress for US-RSE
+tags: [communication]
+---
+
+by Daniel S. Katz, Vanessa Sochat, Charles Ferenbaugh, Jeffrey C. Carver, Sandra Gesing, Ian Cosden, Christina Maimone, and Chris Hill
+
+The US-RSE association was first planned in 2018, but it really became active and started growing in 2019. In the last year or so, we've made a lot of progress and have a number of activities that are in-progress. In this post, we want to report on this progress.
+
+## Accomplishments
+
+**Membership**. In January 2019, we had 10 members, who were all involved in starting up US-RSE. By the end of 2019, this had grown to 259, and is now (at the end of March 2020) 345.
+
+![US-RSE Membership]({{ site.baseurl }}/assets/posts/2020-04-28-us-rse-membership.png "US-RSE Membership")
+
+**Steering Committee**. We've established an initial [steering committee](https://us-rse.org/steering-committee/) with 9 members:
+
+* Jeffrey C. Carver, University of Alabama
+* Ian Cosden, Princeton University (Chair)
+* Charles Ferenbaugh, Los Alamos National Laboratory
+* Sandra Gesing, University of Notre Dame
+* Chris Hill, MIT
+* Daniel S. Katz, University of Illinois Urbana-Champaign
+* Christina Maimone, Northwestern University
+* Lance Parsons, Princeton University
+* Jordan Perr-Sauer, National Renewable Energy Laboratory. 
+
+The Steering Committee is currently meeting every two weeks and working on many of the activities below. In addition, we are planning a formal governance process that will include regular steering committee elections. The initial Steering Committee’s goal is to establish the framework for a long-term vibrant and robust community while defining the overall policy and direction of the US-RSE Association, based on the inputs of the US-RSE Association membership. The Steering Committee encourages and hopes that many new US-RSE Association activities and events will be initiated, organized, and managed by community committees and members at large.
+
+**RSE defined**. We [defined](https://us-rse.org/what-is-an-rse/) Research Software Engineers inclusively, encompassing "all those who regularly use expertise in programming to advance research. This includes researchers who spend a significant amount of time programming, full-time software engineers writing code to solve research problems, and those somewhere in-between. We aspire to apply the skills and practices of software development to research to create more robust, manageable, and sustainable research software."
+
+**Mission**. We've defined an initial mission: The US-RSE Association is centered around three main goals. We aim to target our activities and actions to serving these three aspects:
```

Comment:
Here is the parent section under which "Community," "Advovacy," and "Resources" can be bullet points (or smaller section headers to show child relationship).

Reply:
right, I see - this was my mistake in moving from google docs to markdown

Reply:
fixed

Reply:
This looks great!

![image](https://user-images.githubusercontent.com/814322/80535984-e7f8e100-895e-11ea-994d-79f22dd9a7e8.png)